src/main/java/uk/ac/ebi/eva/pipeline/parameters/ParametersFromProperties.java

```diff
@@ -62,6 +62,12 @@
     @Value(PROPERTY + JobParametersNames.OUTPUT_DIR_STATISTICS + OR_NULL)
     private String outputDirStats;
 
+    @Value(PROPERTY + JobParametersNames.STATISTICS_SKIP + OR_NULL)
+    private String statisticsSkip;
+
+    @Value(PROPERTY + JobParametersNames.ANNOTATION_SKIP + OR_NULL)
```

Comment:
No, they have they are only use on the skip deciders that can't be injected at all with step or job scope and are readed from the context passed to the decide function. This is only the mandatory integration test property to parameters thingy.